src/webgpu/util/conversion.ts

```diff
@@ -2234,6 +2234,13 @@ export function isFloatType(ty: Type): boolean {
   return false;
 }
 
+/**
+ * @returns if `ty` is a floating point type or a composite float type.
+ */
+export function isFloatTypeOrCompositeFloatType(ty: Type): boolean {
```

Comment:
Caller could just use `scalarTypeOf`, but this is fine.

Reply:
I switched to `scalarTypeOf` (thanks!)  And then I could remove the change to `conversions.ts` which means I could remove the new hashes

Reply:
Ace - looks great!